llvm/lib/Transforms/Coroutines/ABI.h

```diff
@@ -0,0 +1,109 @@
+//===- ABI.h - Coroutine ABI Transformers ---------------------*- C++ -*---===//
```

Comment:
I thought the name `ABI Transformers` will transform ABIs to ABIs. But it is not the case right? I feel the name may not necessary here.

Reply:
Thanks for catching this! I had originally used the name Transformers instead of ABI object, so the SwitchABI would be the SwitchTransformer. But that was a bit confusing (as you point out) and in the end it made more sense to drop  'Transformer'. I missed this one...